docs/_adapters/nginx-sm.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: Nginx Servce Mesh Adapter
+name: Meshery Adapter for Nginx Servce Mesh
+mesh_name: Nginx Servce Mesh
+version: v2.5.0
+port: 10001/tcp
+project_status: stable
+github_link: https://github.com/layer5io/meshery-nginx-sm
+image: /docs/assets/img/service-meshes/nginx-sm.svg
+---
+# {{ page.name }}
+
+| Service Mesh   | Adapter Status | Latest Supported Mesh Version |
+| :------------: | :------------:   | :------------:              |
+| {{page.title}} | [{{ page.project_status }}]({{ page.github_link }}) | {{page.version}}  |
```

Comment:
Need to double-check, but I believe this section is now included in the page layout "adapters". // @chandrashritii